index.html

```diff
@@ -0,0 +1,2059 @@
+                <img
+                  class="d-flex mr-3"
+                  alt="Mohammed"
+                  src="https://d.newsweek.com/en/full/1602983/mohammed-alsherebi.png?w=63&h=63&f=f3cb5702ecaac97c55ce6e0f1174eecb"
+                  style="width: 64px; height: 64px;"
+                />
+                <div class="media-body">
+                  <h5 class="mt-0 font-weight-bold">
+                    Could an Israeli-Saudi Peace be Next? Yes, With One Caveat
+                  </h5>
+                  <p class="font-weight-bold">BY MOHAMMED ALSHEREBI</p>
+                </div>
+              </div>
+              <hr />
+              <div class="media">
+                <img
+                  class="d-flex mr-3"
+                  alt="E. Wirth"
+                  src="https://d.newsweek.com/en/full/1604725/tom-rogers-timothy-e-wirth.png?w=63&h=63&f=c9021798f8a0bdb4f9c6625cd060f5ab"
+                  style="width: 64px; height: 64px;"
+                />
+                <div class="media-body">
+                  <h5 class="mt-0 font-weight-bold">
+                    How to Defeat Trump's Plan to Overturn the Election BY
+                    TIMOTHY
+                  </h5>
+                  <p class="font-weight-bold">E. WIRTH AND TOM ROGERS</p>
+                </div>
+              </div>
+              <hr />
+              <div class="media">
+                <img
+                  class="d-flex mr-3"
+                  alt="Ilia"
+                  src="https://d.newsweek.com/en/full/1591871/ilya-shapiro.png?w=63&h=63&f=fe78202c6a864fd4aa4448915c91df8a"
+                  data-holder-rendered="true"
+                />
+                <div class="media-body">
+                  <h5 class="mt-0 font-weight-bold">
+                    Trump Should Add These People to His Supreme Court List
+                  </h5>
+                  <p class="font-weight-bold">BY ILYA SHAPIRO</p>
+                </div>
+              </div>
+              <hr />
+              <div class="media">
+                <img
+                  class="d-flex mr-3"
+                  alt="Michael"
+                  src="https://d.newsweek.com/en/full/1625615/michael-makovsky.png?w=63&h=63&f=d1862ddf967d967845cb2ffb75032dc3"
+                  style="width: 64px; height: 64px;"
+                />
+                <div class="media-body">
+                  <h5 class="mt-0 font-weight-bold">
+                    UAE-Israel Peace Presents Turkey With a Clarifying Moment
+                  </h5>
+                  <p class="font-weight-bold">BY MICHAEL MAKOVSKY</p>
+                </div>
+              </div>
+              <hr />
+              <div class="media">
+                <img
+                  class="d-flex mr-3"
+                  alt="Cliff"
+                  src="https://d.newsweek.com/en/full/1625683/cliff-smith.png?w=63&h=63&f=5cce91c24e19cfadd1e4cd7eb5a6f0ae"
+                  style="width: 64px; height: 64px;"
```

Comment:
- [ ] Please remove the inline styling [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)